coach/src/concept-coach/index.cjsx

```diff
@@ -158,7 +158,7 @@ class ConceptCoachAPI extends EventEmitter2
   updateToView: (view) ->
     if @component
       if view is 'close'
-        @component.props.close()
+        @component.props?.close?()
       else
```

Comment:
a small bug, sometimes it would error when closing if the current component didn't have a handler